public/app/features/transformers/timeSeriesTable/timeSeriesTableTransformer.test.ts

```diff
@@ -51,15 +52,23 @@ describe('timeSeriesTableTransformer', () => {
     expect(results).toHaveLength(2);
     expect(results[0].refId).toBe('A');
     expect(results[0].fields).toHaveLength(3);
-    expect(results[0].fields[0].values).toEqual(['A', 'A', 'A']);
-    expect(results[0].fields[1].values).toEqual(['B', 'C', 'D']);
-    assertDataFrameField(results[0].fields[2], series.slice(0, 3));
+    expect(results[0].fields[0].values).toEqual([
+      'Value : instance=A : pod=B',
+      'Value : instance=A : pod=C',
+      'Value : instance=A : pod=D',
+    ]);
```

Comment:
@codeincarnate this looks like a very strange and big breaking change, the transform created separate fields, and now it's just some value inside a string?